src/m365/aad/commands/m365group/m365group-remove.ts

```diff
@@ -115,6 +109,124 @@ class AadM365GroupRemoveCommand extends GraphCommand {
       }
     }
   }
+
+  private async getM365GroupSiteURL(logger: Logger, id: string): Promise<string> {
+    if (this.verbose) {
+      await logger.logToStderr(`Getting the site URL of Microsoft 365 Group: ${id}...`);
+    }
+
+    const requestOptions: CliRequestOptions = {
+      url: `${this.resource}/v1.0/groups/${id}/drive?$select=webUrl`,
+      headers: {
+        accept: 'application/json;odata.metadata=none'
+      },
+      responseType: 'json'
+    };
+
+    const res = await request.get<{ webUrl: string }>(requestOptions);
+    return res.webUrl.substring(0, res.webUrl.lastIndexOf('/'));
```

Comment:
What's the point of the substring: do we want to trim the trailing `/` or remove the web name from the URL? If it's the former, let's use `.trim()` instead of `substring()`. If it's the latter, let's clarify that with a comment.